packages/server/src/utilities/security.ts

```diff
@@ -22,9 +23,6 @@ export function getPermissionType(resourceId: string) {
     case DocumentType.QUERY:
     case DocumentType.DATASOURCE:
       return permissions.PermissionType.QUERY
-    default:
```

Comment:
Is this not needed for existing views?

Reply:
I've switched over existing views to use the table permission, as it is more accurate now - all of this wasn't baked into anything, it shouldn't affect existing permissions.